src/components/SitePage/SiteData.js

```diff
@@ -27,7 +27,7 @@ class SiteData extends Component {
   }
 
   render() {
-    
+    { this.props.site }
```

Comment:
Note for the mapStateToProps: this is called 'siteReducer' in the reducer index.js. You can write site: state.siteReducer up there, and leave this alone.